docker/bookkeeper/Dockerfile

```diff
@@ -8,11 +8,13 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
 
-FROM openjdk:8-jre
+FROM openjdk:8-jre-alpine
 MAINTAINER Arvind Kandhare [arvind.kandhare@emc.com]
 
-RUN apt-get update \
-&&  apt-get install -y wget unzip \
+RUN apk add --update \
```

Comment:
Do not see the option `--update` inside `apk add -h`. I logged into the container openjdk:8-jre-alpine and ran the apk command.  `apk-get update` is a valid command.

Reply:
It's a shorthand for doing first `apk update` followed by `apk add [package]`. It works fine and many other projects are using it, but as you say, it's not documented...